### PR TITLE
Add UID to relationvalue_converter summary

### PR DESCRIPTION
**Pull request by ksuess**
Support RelationList field with named StaticCatalogVocabulary and SelectWidget

### CODE REVIEW

**Comment by ksuess** [comment]
@jenkins-plone-org please run jobs

**Review by sneridagh — APPROVED** [review]
LGTM

**Comment by sneridagh** [comment]
@tisto could you please take a look and have a release? https://github.com/plone/volto/pull/4614 depends on this one.

**Comment by tisto** [comment]
@sneridagh will do. Need to fix the Python 3.6 thingy in master first.

**Review by tisto — APPROVED** [review]

**Comment by tisto** [comment]
@jenkins-plone-org please run jobs

**Review by tisto — APPROVED** [review]

File: news/1605.feature [hunk 1]
```diff
@@ -0,0 +1 @@
+Add UID to relationvalue_converter summary. @ksuess
```
Comment (tisto) [inline news/1605.feature:1]: @ksuess we use [ksuess] in backend packages. :) Just FYI. I will fix this for you.

Comment (davisagli) [reply news/1605.feature:1]: @tisto We just decided to switch to using `@githubusername` everywhere, for consistency. https://github.com/plone/documentation/pull/1478#discussion_r1157184539

Comment (stevepiercy) [reply news/1605.feature:1]: As a bonus, GitHub renders `@username` format to a clickable username.

Comment (tisto) [reply news/1605.feature:1]: @davisagli ok, that's new for me. Awesome! That makes lots of sense to me. Will stick to this from now on for plone.rest(api) releases.

Comment (tisto) [reply news/1605.feature:1]: @ksuess sorry, for the noise. Turns out you were right and I was wrong. :)

**Pull request merged**